app/monitoringapi.go

```diff
@@ -73,6 +73,24 @@ func wireMonitoringAPI(life *lifecycle.Manager, addr string, localNode *enode.Lo
 	life.RegisterStart(lifecycle.AsyncBackground, lifecycle.StartMonitoringAPI, httpServeHook(server.ListenAndServe))
 	life.RegisterStop(lifecycle.StopMonitoringAPI, lifecycle.HookFunc(server.Shutdown))
 
+	go func() {
+		ticker := time.NewTicker(time.Second)
```

Comment:
we can discuss on the duration part, what will be the right duration to push metrics

Reply:
metrics are not pushed, they are pull. So you do not need to update more than pull period, which is 15s. Suggest making this 10s.